uint32 session_id
uint32 microphone_id
uint32 csound_id
time ts
int16[] audio